dbt_subprojects/daily_spellbook/models/evms/evms_info.sql

```diff
@@ -104,6 +104,6 @@ FROM (
         , (57073, 'ink', 'Ink', 'Layer 2', 'Optimistic Rollup', 'ETH', 0x4200000000000000000000000000000000000006, 'https://explorer.inkonchain.com/', timestamp '2024-12-06 15:20', 'OP Stack', 'Ethereum Blobs', 'Ethereum', true)
         , (2741, 'abstract', 'Abstract', 'Layer 2', 'ZK Rollup', 'ETH', 0x3439153EB7AF838Ad19d56E1571FBD09333C2809, 'https://abscan.org/', timestamp '2024-10-25 18:31', 'ZK Stack', 'Ethereum Blobs', 'Ethereum', true)
         , (80094, 'berachain', 'Berachain', 'Layer 1', NULL, 'BERA', NULL, 'https://berascan.com/', timestamp '2024-01-20 14:14:11', NULL, NULL, NULL, true)
+        , (33139, 'apechain', 'ApeChain', 'Layer 3', 'Optimistic Rollup', 'APE', NULL, 'https://apescan.io/', timestamp '2024-02-19 00:00', 'Arbitrum Nitro', 'AnyTrust', 'Arbitrum One', true)
```

Comment:
L3 ...?
https://docs.apechain.com/architecture

Reply:
yep, i guess so 🤷 first one of many?